backend/api/migrations/0204_auto_20230316_2115.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 3.2.18 on 2023-03-16 21:15
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('api', '0203_auto_20230312_0823'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
```

Comment:
are we sure there isn't any information we're losing with this removal?

Reply:
We would need to manually update DB to append Address_line_3 data with Address_line_2 and county content with Province unless there is better way to do it. In a local i did update DB manually with query to make sure we don't lose data.

Reply:
@jig-patel please update the based branch to release-2.5.0 as 2.4.0 has been released to prod

Reply:
@kuanfandevops thanks. will do